packages/core/src/saml-applications/libraries/saml-applications.ts

```diff
@@ -86,7 +92,7 @@ export const createSamlApplicationsLibrary = (queries: Queries) => {
       Object.keys(applicationData).length > 0
         ? updateApplicationById(id, removeUndefinedKeys(applicationData))
         : originalApplication,
-      config
+      Object.keys(config).length > 0
```

Comment:
Will this partially update the config values?  E.g. { entityId: XXX }

Reply:
Add integration test to cover this.